featurebyte/models/base.py

```diff
@@ -20,6 +21,7 @@
 
 DEFAULT_CATALOG_ID = ObjectId("63eda344d0313fb925f7883a")
 ACTIVE_CATALOG_ID: ObjectId = DEFAULT_CATALOG_ID
+CAMEL_CASE_TO_SNAKE_CASE_PATTERN = re.compile("((?!^)(?<!_)[A-Z][a-z]+|(?<=[a-z0-9])[A-Z])")
```

Comment:
nit: can consider sharing the pattern / function with the app_container_config

Reply:
oh nvm it is shared, sorry i missed it.

Reply:
This pattern is moved from `app_container_config`. At the app container module, it will import the pattern from here.